comid/integrityregisters.go

```diff
@@ -18,11 +18,11 @@ type IRegisterIndex interface{}
 
 // IntegrityRegisters holds the Integrity Registers
 type IntegrityRegisters struct {
-	m map[IRegisterIndex]Digests
+	M map[IRegisterIndex]Digests
```

Comment:
Why is this now being exported (this does not appear to be referenced outside the `comid` package by these changes).

If this does need to be accessible exteranlly, perhaps a more meaningful name than `M` (e.g `Digests` or `DigestMap` or something)?

Reply:
Yes, this does needs external access, hence the change.

I am not fussy about names, for brevity it was kept like that and it is a register entry, not a Digests as such!